src/components/admin/order/ManageOrder.jsx

```
@@ -558,19 +597,19 @@ export default function ManageOrder() {
+                filteredOrders.map((order) => (
+                  <OrderRow
+                    key={order == null ? 0 : order.orderId}
+                    row={order}
+                    orderStatus={orderStatus}
+                    updateOrderStatusBySelect={updateOrderStatusBySelect}
+                    updateOrderStatusByClick={updateOrderStatusByClick}
+                    openDocumentModal={openDocumentModal}
+                    openReportModal={openReportModal}
+                    cancelOrder={cancelOrder}
+                  />
+                ))}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)